ONSdigital/ssdc-rm-response-operations
# Updating dependabot dependencies

# Motivation and Context

There's a bunch of dependabot PRs that need to be addressed. I've put some onto a single PR to fully test it as some were failing silently.
# What has changed



- Updated the dependencies
# How to test?



- Run make build and run with the ATs to make sure it all passes
- Check the page to make sure nothing looks broken
# Links



[Jira](https://jira.ons.gov.uk/browse/SDCSRM-463)
# Screenshots (if appropriate):
